src/tests/EdFi.DataManagementService.Tests.E2E/Features/General/DiscoveryAPI.feature.disabled

```diff
@@ -0,0 +1,66 @@
+Feature: The Discovery API provides information about the application version, supported data model(s), and URLs for additional metadata.
+
+        @ignore
+        Scenario: 01 GET / returns the root Discovery API document
+            Given a GET to the root of the API
+             Then it should respond with 200
+              And the response body is
+                  """
+                  {
+                      "version": "1.0.0",
+                      "applicationName": "Ed-Fi Alliance Data Management Service"
+                      "dataModels": [
+                          {
+                              "name": "Ed-Fi",
+                              "version": "5.1.0",
+                              "informationalVersion": "The Ed-Fi Data Standard v5.1"
+                          }
+                      ],
+                      "urls": [
+                          "dependencies": "{BASE_URL}/metadata/dependencies",
+                          "openApiMetadata": "{BASE_URL}/DMS/metadata/specifications",
+                          "oauth": "{BASE_URL}/DMS/oauth/token",
+                          "dataManagementApi": "{BASE_URL}/DMS/data",
+                          "xsdMetadata": "{BASE_URL}/DMS/metadata/xsd",
+                          "changeQueries": "{BASE_URL}/DMS/changeQueries/v1",
+                          "identity": "{BASE_URL}/DMS/identity/v2"
+                      ]
+                  }
+                  """
+
+        @ignore
+        Scenario: 02 GET /metadata returns the metadata URL list
+            Given a GET to "/metadata"
+             Then it should respond with 200
+              And there is a JSON file with the list of metadata URL
```

Comment:
This one is simple and we can be explicit about the response body.

```json
{
    "dependencies": "{BASE_URL}/metadata/dependencies",
    "specifications": "{BASE_URL}/metadata/specifications",
    "xsdFiles": "{BASE_URL}/metadata/xsdMetadata"
}
```

Reply:
Response body updated